docs/tools/vega-wallet/index.md

```diff
@@ -3,7 +3,9 @@ title: Vega Wallet
 hide_title: false
 ---
 
-Vega Wallet allows you to manage wallets and key pairs, and sign transactions. You can interact with a Vega wallet and its keys through two different apps: 
+Vega Wallet allows you to manage wallets and key pairs, and sign transactions.
```

Comment:
Suggestion: Vega Wallet allows you to manage wallets and key pairs, sign and **broadcast** transactions.